.changeset/empty-fireants-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend-module-vmware-cloud-provider': patch
```

Comment:
I think I remember @freben saying the convention for the first version was 0.1.0
```suggestion
'@backstage/plugin-auth-backend-module-vmware-cloud-provider': minor
```

Reply:
Yep, not critical but a pretty good convention